### PR TITLE
QA-15101 Remove manual registration of JCR Event listeners.

### PR DESCRIPTION
**Pull request by jayblanc**
## JIRA

https://jira.jahia.org/browse/QA-15101

## Description

Upgrade samples to the latest module parent
Due to modifications in the way JCREventListener are registered, there is no more need to do it manually.

## Tests

The following are included in this PR
- [ ] Unit Tests (Most changes _should_ have unit tests)
- [ ] Integration Tests

## Checklist

I have considered the following implications of my change: 

- [X] Security (in particular for changes to authentication, authorization, data fetching, ...)
- [X] Performance
- [X] Migration
- [X] Code maintainability

## Documentation

- [ ] Inline documentation
- [ ] Internal Documentation (wiki)
- [ ] User-facing Documentation

### CODE REVIEW

File: servlet-filter-samples/pom.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <parent>
         <artifactId>jahia-modules</artifactId>
         <groupId>org.jahia.modules</groupId>
-        <version>8.0.0.0</version>
```
Comment (dgriffon) [inline servlet-filter-samples/pom.xml:10]: You need `<relativePath/>` as for other modules

File: login-provider/pom.xml [hunk 2]
```diff
@@ -7,8 +7,7 @@
     <parent>
         <artifactId>jahia-modules</artifactId>
         <groupId>org.jahia.modules</groupId>
-        <version>8.0.3.0</version>
-        <relativePath />
```
Comment (dgriffon) [inline login-provider/pom.xml:11]: You need `<relativePath/>` as for other modules

**Review by dgriffon — CHANGES_REQUESTED** [review]
As a sample code, it would be better to have as a parent module the released version of Jahia

**Comment by jayblanc** [comment]
replaced by https://github.com/Jahia/OSGi-modules-samples/pull/36

**Pull request closed without merging**